features/ouders.feature

```diff
@@ -19,236 +19,234 @@ Rule: De ouder in categorie 2 heeft ouderAanduiding "1" en de ouder in categorie
-    Gegeven de persoon met burgerservicenummer 999993008 heeft de volgende ouders in de registratie
-    | Categorie | Voornamen (02.10)  | Geslachtsnaam | Geboortedatum (03.10) | Geslachtsaanduiding (04.10) | Datum ingang familierechtelijke betrekking (62.10) |
-    | 2         |                    | .             |                       | V                           | 20190614                                           |
-    | 3         |                    |               |                       |                             |                                                    |
-    En in categorie 3 komt alleen groep 81 en groep 85 voor
+    Gegeven de persoon met burgerservicenummer '000007896' heeft een ouder '1' met de volgende gegevens
+    | datum ingang familierechtelijke betrekking (62.10) | geslachtsnaam (02.40) | geslachtsaanduiding (04.10) |
+    | 20190614                                           | .                     | V                           |
+    En de persoon heeft een ouder '2' met de volgende gegevens
+	| aktenummer (81.20) | ingangsdatum geldigheid (85.10) |
+	| 2•E0001            | 20160518                        |
     Als personen wordt gezocht met de volgende parameters
     | naam                | waarde                          |
     | type                | RaadpleegMetBurgerservicenummer |
-    | burgerservicenummer | 999993008                       |
+    | burgerservicenummer | 000007896                       |
     | fields              | ouders                          |
-    Dan bevat de persoon alleen ouders met de volgende gegevens
-    | naam                                    | waarde   |
-    | ouderAanduiding                         | 1        |
-    En heeft de ouder met ouderAanduiding '1' de volgende gegevens
-    | naam                                    | waarde   |
-    | ouderAanduiding                         | 1        |
-    | geslacht.code                           | V        |
-    | geslacht.omschrijving                   | vrouw    |
-    | datumIngangFamilierechtelijkeBetrekking | 20190614 |
-    En de ouder met ouderAanduiding '1' heeft GEEN 'geboorte' gegevens
-    En heeft de ouder met ouderAanduiding '1' alleen de volgende 'naam' gegevens
-    | naam          | waarde |
-    | geslachtsnaam | .      |
+    Dan heeft de response een persoon met een 'ouder' met de volgende gegevens
+    | naam                                                | waarde       |
+    | ouderAanduiding                                     | 1            |
+    | geslacht.code                                       | V            |
+    | geslacht.omschrijving                               | vrouw        |
+    | datumIngangFamilierechtelijkeBetrekking.type        | Datum        |
+    | datumIngangFamilierechtelijkeBetrekking.datum       | 2019-06-14   |
+    | datumIngangFamilierechtelijkeBetrekking.langFormaat | 14 juni 2019 |
```

Comment:
er moet ook een leeg naam object worden opgenomen. Dat staat nu niet beschreven. Als het goed is faalt dit scenario daarop.

Reply:
Zie mijn opmerking elders in deze PR.